src/backend/aspen/cli/db.py

```diff
@@ -408,3 +410,54 @@ def create_phylo_run(
 
         workflow_id = workflow.workflow_id
     print(workflow_id)
+
+
+@db.command("create-mega-fasta")
+@click.option(
+    "public_identifier_input_fh",
+    "--public-identifier-txt",
+    type=click.File("r"),
+    required=True,
+)
+@click.option(
+    "sequences_output_fh", "--sequences-output", type=click.File("w"), required=True
+)
+@click.pass_context
+def create_mega_fasta(
+    ctx, public_identifier_input_fh: io.TextIOBase, sequences_output_fh: io.TextIOBase
+):
+
+    engine = ctx.obj["ENGINE"]
+    session = engine.make_session()
+
+    public_identifiers: Collection[str] = [
+        line.strip() for line in public_identifier_input_fh
+    ]
+
+    all_samples: Iterable[Sample] = (
+        session.query(Sample)
+        .filter(Sample.public_identifier.in_(public_identifiers))
+        .options(
+            joinedload(Sample.uploaded_pathogen_genome).undefer(
+                UploadedPathogenGenome.sequence
+            ),
+        )
+    )
+
+    # TODO: this code is now used in multiple places, find central location to store this
+    for sample in all_samples:
+        # filter for samples that passed genome recovery
```

Comment:
consider a TODO: for supporting called pathogen genomes (or an explanation of why it's not necessary)

Reply:
ok i will add a todo for now